### ISSUE TITLE
Blenderbim IfcSite attribute edition : Error when validating Lattitude and Longitude

### ISSUE THREAD

**Issue by Gorgious56** [IfcOpenShell/IfcOpenShell#3128]
Hehe so it begins. :)

This bug report is twofold :
- When trying to validate RefLattitude and RefLongitude of my IfcSite which I (maybe wrongly) logically assumed to be numbers in the format 48.455884, -3.508734, an error is thrown.

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/25156105/8e3bc30e-b60a-43fb-8507-a29d0ce6b431)

```
Traceback (most recent call last):
  File "C:\Users\NH\AppData\Roaming\Blender Foundation\Blender\3.5\scripts\addons\blenderbim\bim\module\attribute\operator.py", line 32, in execute
    IfcStore.execute_ifc_operator(self, context)
  File "C:\Users\NH\AppData\Roaming\Blender Foundation\Blender\3.5\scripts\addons\blenderbim\bim\ifc.py", line 410, in execute_ifc_operator
    result = getattr(operator, "_execute")(context)
  File "C:\Users\NH\AppData\Roaming\Blender Foundation\Blender\3.5\scripts\addons\blenderbim\bim\module\attribute\operator.py", line 116, in _execute
    ifcopenshell.api.run("attribute.edit_attributes", self.file, product=product, attributes=attributes)
  File "c:\Program Files (x86)\Steam\steamapps\common\Blender\3.5\python\lib\site-packages\ifcopenshell\api\__init__.py", line 66, in run
    result = usecase_class(ifc_file, **settings).execute()
  File "c:\Program Files (x86)\Steam\steamapps\common\Blender\3.5\python\lib\site-packages\ifcopenshell\api\attribute\edit_attributes.py", line 52, in execute
    setattr(self.settings["product"], name, value)
  File "c:\Program Files (x86)\Steam\steamapps\common\Blender\3.5\python\lib\site-packages\ifcopenshell\entity_instance.py", line 239, in __setattr__
    self[index] = value
  File "c:\Program Files (x86)\Steam\steamapps\common\Blender\3.5\python\lib\site-packages\ifcopenshell\entity_instance.py", line 267, in __setitem__
    self.method_list[idx](
  File "c:\Program Files (x86)\Steam\steamapps\common\Blender\3.5\python\lib\site-packages\ifcopenshell\ifcopenshell_wrapper.py", line 4908, in setArgumentAsAggregateOfInt
    return _ifcopenshell_wrapper.entity_instance_setArgumentAsAggregateOfInt(self, i, v)
TypeError: Attribute of type AGGREGATE OF INTEGER needs a python sequence of ints
```

- After a bit of forehead scratching I inferred that the parser is looking for values separated with commas, so I typed the values with commas to a (seemingly) successful attribute validation. Weird, but OK

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/25156105/d5dc9a93-6781-4c5a-893e-096f91135193)

Alas, when bringing back the attribute edition state the values are now lost to oblivion.

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/25156105/3ec30556-d6f6-4e3f-917f-4534c7efd4cc)

And my handy right click in the field > IfcDescription doesn't even work for this field, so I can't even go looking online for what it's supposed to contain.

**Comment by Moult** [comment]
Try enter in a JSON string of degrees, minutes, seconds

**Comment by Gorgious56** [comment]
Oh okay. What would be the correct syntax ? I tried `{"degrees":"48", "minutes":"27", "seconds":"1824"}` but failed

**Comment by Moult** [comment]
At a guess [123,456,789] ?

**Comment by Gorgious56** [comment]
Thanks, it does work with this format ! However it doesn't accept decimal places for seconds, that's a bit unfortunate. 
I don't think it's a problem to have to input it that way, but the user should be advised somehow. After a bit of digging I realized you implemented a special case for these specific values. So I went ahead and simply added the IFC description which I noticed were missing and added an extra line to tell the user what they should do.
![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/25156105/e33ec9e8-f980-4a6a-8f5b-92d7afa54264)

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/25156105/1e2a771d-808d-4d3e-823a-624726812e38)

As far as I'm concerned this issue can be closed if you feel like my solution is OK. However I'm all ears for suggestions to how make it better :)

Cheers
